src/plugins/intel_gna/src/transformations/utils/transformation_helper.cpp

```diff
@@ -13,6 +13,29 @@ namespace intel_gna {
 namespace pass {
 namespace helper {
 
+void GetConvData(std::shared_ptr<ngraph::op::ConvolutionIE> conv, ConvData& conv_data) {
```

Comment:
Could you add the description here, and why we are using c-style return way? If we updating the conv_data let's name it UpdateConvData or SetConvDate

Reply:
Fixed `GetConvData` -> `UpdateConvData`